docs/site/src/components/ApiMenu.js

```diff
@@ -0,0 +1,68 @@
+// @flow weak
+
+import React, { PureComponent, PropTypes } from 'react';
+import { camelCase } from 'docs/site/src/utils/helpers';
+import IconButton from 'material-ui/IconButton';
+import { Menu, MenuItem } from 'material-ui/Menu';
+import MoreVertIcon from 'material-ui/svg-icons/more-vert';
+
+
+export default class ApiMenu extends PureComponent {
```

Comment:
> Class inheritance eww. Function composition yahh.

Wait, what? As a stateful compenent we have to inherit one way or the other, so why inherit `Component` and wrap with `pure` when you can inherit `PureComponent`?

Better to save `recompose/pure` for SFCs, no?